github/create-release.ps1

```diff
@@ -0,0 +1,86 @@
+<#
+.SYNOPSIS
+Trigger runs on the workflow_dispatch event to create tool release
+
+.PARAMETER RepositoryFullName
+Required parameter. The owner and repository name. For example, 'actions/versions-package-tools'
+.PARAMETER AccessToken
+Required parameter. PAT Token to authorize
```

Comment:
nitpick:
PAT - Personal Access Token
so it looks like Personal Access Token Token to authorize. Please, remove second "Token"

Reply:
Thank you!